core/node/eth_sender/README.md

```diff
@@ -0,0 +1,34 @@
+# Eth-sender `zksync_eth_sender`
+
+Eth-sender is the component that executes scheduled transactions on Ethereum. As of 06.2024 it is only used by core to
+execute Commit, Prove and Execute L1 transactions.
+
+It was designed with fire-and-forget ideology in mind, as soon as you send transaction to be executed, eth-sender will
+make sure it is mined, handling stuff as bumping gas prices and resending transaction if needed.
+
+## API
+
+eth-sender API exposes only two methods:
+
+```
+send_tx(idempotency_key, raw_tx, tx_type, contract_address, blob_sidecar) -> None
+```
+
+```
+get_tx(idempotency_key) -> Optional<(tx_hash, status)>
+```
+
+where `idempotency_key` can be just any String and status be one of `Pending`, `Confirmed`, `Failed`
+
+Attempting to send tx for the same `idempotency_key`, but with different parameters returns in an error.
+
+Nonces are determined via order of calling send_tx.
+
+## Integration with server-v2
+
+Whenever we want to send an L1 transaction, we create an entry in `l1_transactions` table with status `Created`. We have
+eth_sender_sync that periodically picks up all entries from l1_transactions. It finds all that have status `Created` and
```

Comment:
this new 'component' will belong to server-v2, either as part of housekeeper or as a separate component. @RomanBrodetski do you have preference whether it needs to be separate? It will be relatively small piece of code with practically no config